### ISSUE TITLE
Please add user-select:none on the `<canvas>` element.

### ISSUE THREAD

**Issue by Francisc** [subprotocol/verlet-js#6]
When you play with the mouse, you select the canvas.
Only tested on Canary 28 on Windows 7.

``` CSS
-webkit-user-select:none;
-moz-user-select:-moz-none;
-o-user-select:none;
-ms-user-select:none;
-khtml-user-select:none;
user-select:none;
```

**Comment by subprotocol** [comment]
User-select: none should be in there, but I have not updated and pushed the sub module for the site.  I am traveling and can't do this right now.

Would you mind checking out master and verifying that it no longer exhibits the issue?

Thanks!

**Comment by subprotocol** [comment]
This is now fixed by 843f451d7c06ff19210b4b326da5ab41f5381855

Thanks for the heads up.
